app/models/katello/gpg_key.rb

```diff
@@ -1,5 +1,8 @@
 module Katello
   class GpgKey < Katello::Model
+    audited
+    audit_associations :products
```

Comment:
you can combine both in one as `audited :associations => :products`, `audit_associations` is mostly needed when we want to add associations to an existing model.